src/pages/warehouse/all-warehouses/WarehousesPage.tsx

```diff
@@ -69,18 +77,26 @@ function Warehouses() {
                 <TableCell>Warehouse name</TableCell>
                 <TableCell align="right">Address</TableCell>
                 <TableCell align="right">Phone</TableCell>
+                <TableCell align="right">Admin</TableCell>
                 <TableCell align="right">Area</TableCell>
               </TableRow>
             </TableHead>
             <TableBody>
-              {warehouses.map((warehouse) => (
-                <TableRow key={warehouse.id}>
+              {warehouses.map((ware) => (
+                <TableRow key={ware.warehouse.id}>
                   <TableCell component="th" scope="row">
-                    <Link to={`${warehouse.id}`}>{warehouse.name}</Link>
+                    <Link to={`${ware.warehouse.id}`}>
+                      {ware.warehouse.name}
+                    </Link>
+                  </TableCell>
+                  <TableCell align="right">{ware.warehouse.address}</TableCell>
+                  <TableCell align="right">{ware.warehouse.phone}</TableCell>
+                  <TableCell align="right">
+                    <Link to={`/users/${ware.user.id}`}>
+                      {ware.user.first_name} {ware.user.last_name}
+                    </Link>
                   </TableCell>
```

Comment:
Backend response is Array, for now - with only one user. So, you need to ware.user[0].id (the same for all data in user)